torch/linalg/__init__.py

```diff
@@ -1654,7 +1660,18 @@
 triangular part of the matrix is used in the computations.
 
 The singular values (or the norm of the eigenvalues when :attr:`hermitian`\ `= True`)
-that are below the specified :attr:`rcond` threshold are treated as zero and discarded in the computation.
+that are below :math:`\max(\text{atol}, \sigma_1 * \text{rtol})` threshold are
+treated as zero and discarded in the computation,
+where :math:`\sigma_1` is the largest singular value (or eigenvalue).
```

Comment:
Is this line correct? If so, we should use the squared norm of the eigenvalue of largest norm, to be consistent between the two definitions.